build.gradle

```
@@ -4,7 +4,7 @@ plugins {
     id 'maven-publish'
     id 'signing'
     id 'io.github.gradle-nexus.publish-plugin' version '1.3.0'
-    id 'com.adarshr.test-logger' version '4.0.0'
+//    id 'com.adarshr.test-logger' version '4.0.0'
```

```suggestion
    id 'com.adarshr.test-logger' version '4.0.0'
```
Let's uncomment this, I did it to verify that weird output isn't caused by it, but forgot to turn it back